parsers/EIA.py

```diff
@@ -477,21 +499,21 @@ def fetch_exchange(
     return exchange
 
 
-def _fetch_series(
+def _fetch(
     zone_key: str,
-    series_id,
+    url_prefix: str,
     session: Optional[Session] = None,
     target_datetime: Optional[datetime] = None,
     logger: Logger = getLogger(__name__),
 ):
-    """Fetches and converts a data series."""
-    s = session or Session()
-
-    # local import to avoid the exception that happens if EIAPY token is not set
-    # even if this module is unused
-    from eiapy import Series
```

Comment:
If [`eiapy`](https://github.com/systemcatch/eiapy) is no longer used it should be removed from the project all together.

>**Note**
>This can be done by running `poetry remove eiapy`.

Reply:
Thanks for reminding!